app/models/intake.rb

```diff
@@ -207,7 +207,6 @@ class Intake < ApplicationRecord
   belongs_to :client, inverse_of: :intake, optional: true
   has_many :tax_returns, through: :client
   belongs_to :vita_partner, optional: true
-  belongs_to :triage_source, optional: true, polymorphic: true
```

Comment:
Just to quick check, Intakes can't have any other kind of triage source?

Reply:
There are no other triage sources -- seems like the original intention was that there would be since it was set up to be polymorphic, but we don't use triage_source in our current implementation of triaging, either.